content/200-concepts/100-components/01-prisma-schema/07-names-in-underlying-database.mdx

```diff
@@ -141,7 +141,7 @@ Since no explicit names are provided via `map` arguments Prisma will assume they
 
 If you introspect a database the names for indexes and constraints will be added to your schema unless they follow Prisma's naming convention. If they do, the names are not rendered to keep the schema more readable. When you migrate such a schema Prisma will infer the default names and persist them in the database.
 
-We chose our naming convention to align with Postgre s since it is deterministic and helps us maximize the amount of times where we do not need
+We chose our naming convention to align with Postgres since it is deterministic and helps us maximize the amount of times where we do not need
```

Comment:
Let's fix it to the correct term at the same time:
```suggestion
We chose our naming convention to align with PostgreSQL since it is deterministic and helps us maximize the amount of times where we do not need
```

Reply:
That works too 😃

Reply:
Optimally you would accept this suggestion, or adapt your PR manually :D

Reply:
Whatever works best for you.